src/memorial_logger.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+#ifndef CATA_MEMORIAL_LOGGER_H
+#define CATA_MEMORIAL_LOGGER_H
+
+#include <map>
+#include <string>
+#include <vector>
+
+#include "event_bus.h"
+
+class memorial_logger : public event_subscriber
+{
+    public:
+        void clear();
+
+        void add_memorial_log( const std::string &/*male_msg*/,
+                               const std::string &/*female_msg*/ );
+        template<typename ...Args>
+        void add_memorial_log( const char *const male_msg, const char *const female_msg, Args &&... args ) {
```

Comment:
The names of these functions can be shorted to just `add_log` or even just `add`. The context (being called on an object of type `memorial_logger`) makes it clear what it adds. Note that the full name of those functions is currently `memorial_logger::add_memorial_log`, which is quite repetitive.

Reply:
Good point.  I shortened some of the other function names too.